pkg/component/coredns/monitoring.go

```diff
@@ -89,6 +89,8 @@ honor_labels: false
 kubernetes_sd_configs:
 - role: endpoints
   api_server: https://` + v1beta1constants.DeploymentNameKubeAPIServer + `:` + strconv.Itoa(kubeapiserverconstants.Port) + `
+  namespaces:
+    names: [ kube-system ]
```

Comment:
Shouldn't we add the

```
follow_redirects: false
```

also in 1e628f2fe39929a6650ec770a4d1573cbeeb1aa1, like in the previous commit?